### PR TITLE
[devops] Add new  network security policies for OpenShift

### PR DESCRIPTION
**Pull request by fwpushan**

### CODE REVIEW

File: devops/openshift/networkpolicy.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    metadata:
+      name: allow-same-namespace
+    spec:
+      # Allow all pods within the current namespace to communicate
+      # to one another.
+      podSelector:
+      ingress:
+      - from:
+        - podSelector: {}
+  - apiVersion: security.devops.gov.bc.ca/v1alpha1
```
Comment (will-freshworks) [inline devops/openshift/networkpolicy.yml:47]: Do we still need these two NSPs?

File: devops/Makefile [hunk 2]
```diff
@@ -127,7 +127,7 @@ print-status:
 
 init-project:
 	@echo "Run initial set-up for the project, including network security\n" 
-	@oc process -f openshift/security-init.yml -p NAMESPACE=$$(oc project --short) | oc apply -f -
```
Comment (will-freshworks) [inline devops/Makefile:130]: We should probably delete the old `security-init.yml` file too

**Review by will-freshworks — CHANGES_REQUESTED** [review]

File: devops/Makefile (hunk 2 above)
Comment (fwpushan) [reply devops/Makefile:130]: I just used the file mentioned in [here](https://developer.gov.bc.ca/Networkpolicy-Migration-Workshop ) and [here](https://github.com/bcgov/networkpolicy-migration-workshop/blob/main/quickstart.yaml)

File: devops/openshift/networkpolicy.yml (hunk 1 above)
Comment (fwpushan) [reply devops/openshift/networkpolicy.yml:47]: I just used the file mentioned in [here](https://developer.gov.bc.ca/Networkpolicy-Migration-Workshop ) and [here](https://github.com/bcgov/networkpolicy-migration-workshop/blob/main/quickstart.yaml)

**Comment by will-freshworks** [comment]
Happy for this to be merged once the old security-init.yml file is deleted.

File: devops/Makefile (hunk 2 above)
Comment (fwpushan) [reply devops/Makefile:130]: Deleted

**Pull request merged**